### ISSUE TITLE
只使用dw conv的实验结果

### ISSUE THREAD

**Issue by Asthestarsfalll** [LeapLabTHU/FLatten-Transformer#3]
感谢你们出色的工作！注意到加了dw conv之后性能有十分显著的提升，请问有单独使用dw conv实验的结果吗？这篇[工作](https://arxiv.org/abs/2106.04263v1)中提到将注意力模块替换为dw conv可以达到与原本相近甚至更好的效果，所以我很好奇dw conv在flatten中所起的作用。

**Comment by tian-qing001** [comment]
Hi @Asthestarsfalll, we genuinely value your interest in and recognition of our work. I'd like to inquire if you were specifically referring to using vanilla linear attention + depthwise convolution without incorporating the focusing function?

**Comment by Asthestarsfalll** [comment]
Hi @tian-qing001, thank you for your reply.
What I mean is replacing vanilla attention with depthwise convolution. At first I'm curious about which portion is dominant in flatten. But upon reviewing the paper I mentioned above, I realized that I had misremembered the details. In addition to replacing vanilla attention with dwconv, the paper also includes some extra layers.

![2023-09-04_12-32](https://github.com/LeapLabTHU/FLatten-Transformer/assets/72954905/9f52a868-69f0-46a7-98f5-fb4b34964608)
